diff -rupN Amble 2.xcodeproj/project.pbxproj Amble 3.xcodeproj/project.pbxproj
--- Amble 2.xcodeproj/project.pbxproj	2015-04-13 12:37:43.000000000 +1000
+++ Amble 3.xcodeproj/project.pbxproj	2015-04-13 12:48:16.000000000 +1000
@@ -927,18 +927,17 @@
 				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
 				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
-				INFOPLIST_FILE = "Amble WatchKit Extension/Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/../../../../../.cordova-build/extensions/watch/Amble Watchkit Extension/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.2;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @executable_path/../../Frameworks";
 				LIBRARY_SEARCH_PATHS = (
 					"$(inherited)",
-					"/Users/timhingston/Documents/dev/percolate/amble/.cordova-build/extensions/watch/Amble",
-					Watchkit,
-					Extension/uk.co.ilee.applewatch,
+					"$(SRCROOT)/Amble/Plugins/uk.co.ilee.applewatch",
 				);
 				MTL_ENABLE_DEBUG_INFO = YES;
 				PRODUCT_NAME = "${TARGET_NAME}";
 				SKIP_INSTALL = YES;
+				SWIFT_OBJC_BRIDGING_HEADER = "$(SRCROOT)/../../../../../.cordova-build/extensions/watch/Amble WatchKit Extension/Amble WatchKit Extension-Bridging-Header.h";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 			};
 			name = Debug;
@@ -960,18 +959,17 @@
 				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
 				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
-				INFOPLIST_FILE = "Amble WatchKit Extension/Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/../../../../../.cordova-build/extensions/watch/Amble Watchkit Extension/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.2;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @executable_path/../../Frameworks";
 				LIBRARY_SEARCH_PATHS = (
 					"$(inherited)",
-					"/Users/timhingston/Documents/dev/percolate/amble/.cordova-build/extensions/watch/Amble",
-					Watchkit,
-					Extension/uk.co.ilee.applewatch,
+					"$(SRCROOT)/Amble/Plugins/uk.co.ilee.applewatch",
 				);
 				MTL_ENABLE_DEBUG_INFO = NO;
 				PRODUCT_NAME = "${TARGET_NAME}";
 				SKIP_INSTALL = YES;
+				SWIFT_OBJC_BRIDGING_HEADER = "$(SRCROOT)/../../../../../.cordova-build/extensions/watch/Amble WatchKit Extension/Amble WatchKit Extension-Bridging-Header.h";
 				VALIDATE_PRODUCT = YES;
 			};
 			name = Release;
@@ -1001,7 +999,7 @@
 				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				IBSC_MODULE = Amble_WatchKit_Extension;
-				INFOPLIST_FILE = "Amble WatchKit App/Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/../../../../../.cordova-build/extensions/watch/Amble Watchkit App/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.2;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				PRODUCT_NAME = "$(TARGET_NAME)";
@@ -1030,7 +1028,7 @@
 				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				IBSC_MODULE = Amble_WatchKit_Extension;
-				INFOPLIST_FILE = "Amble WatchKit App/Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/../../../../../.cordova-build/extensions/watch/Amble Watchkit App/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.2;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				PRODUCT_NAME = "$(TARGET_NAME)";
Binary files Amble 2.xcodeproj/project.xcworkspace/xcuserdata/timhingston.xcuserdatad/UserInterfaceState.xcuserstate and Amble 3.xcodeproj/project.xcworkspace/xcuserdata/timhingston.xcuserdatad/UserInterfaceState.xcuserstate differ
